server/src/routes/api/v1/reviewsVotesRouter.js

```diff
@@ -0,0 +1,28 @@
+import express from "express"
+import { Vote, Review } from "../../../models/index.js"
+import VoteSerializer from "../../../serializers/VoteSerializer.js"
+import objection from "objection"
+const {ValidationError} = objection
+
+const reviewsVotesRouter = new express.Router({ mergeParams: true })
+
+reviewsVotesRouter.post("/", async (req, res) => {
+    const body = req.body
+    body.reviewId = req.params.reviewId
+    body.userId = req.user.id
+    try {
+      const newVote = await Vote.addVote(body)
+      const review = await Review.query().findById(body.reviewId)
+      const votes = await review.$relatedQuery("votes")
+      const newVoteCount = await VoteSerializer.getSummary(votes)
```

Comment:
we shouldnt need to get all of the votes here. instead, we can update the vote count on the frontend and just return the newly created votes

because it is the last day, I don't actually want you to make that change (it would take too long) but something you should consider in BTs